extensions/amp-accordion/0.1/amp-accordion.css

```diff
@@ -29,3 +29,13 @@
   padding-right: 20px;
   border: solid 1px #dfdfdf;
 }
+
+amp-accordion > section:not([expanded]) > :last-child[rendersubtree] {
+  display: block !important;
+  width: 100%;
```

Comment:
I think all of these have to be `!important`. Width is actually the least problematic, but definitely `margin`, `padding`, etc.